stylesheet.css

```diff
@@ -0,0 +1,44 @@
+body{
```

Comment:
Nice use of CSS. One feature you may want to turn on is `Format on Save`. It's an incredibly helpful setting in VS Code.